### PR TITLE
Include namespace in function creation status hint if nondefault

### LINKED ISSUE #720: Status hint should include namespace
When not using `--wait` or `--verbose`, we print something in the lines of
```
Issue `riff service status foobar` to see the status of the function
```

When the service is created in a non default namespace, that hint should include that as well

### PR DESCRIPTION
**Pull request by fbiville**
Fixes #720.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@fbiville Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=projectriff/riff&pullRequestId=735)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=projectriff/riff&pullRequestId=735) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by fbiville** [comment]
Adding a test only now, as I first (mis)understood the change was not easily testable.

**Comment by jldec** [comment]
Please add a description at the top, and link to issue.

**Comment by fbiville** [comment]
@jldec I just added the issue number in the description
do you think the commit title and linked issue reference are now enough to understand the commit or shall I add a description?

**Comment by jldec** [comment]
@fbiville - the link back to the issue which has a reasonable description should be enough, unless you want to call out the specific design/text implemented in the PR here.

**Comment by fbiville** [comment]
Thanks for the quick response.
I believe the commit contains enough information now.

File: cmd/commands/function_test.go [hunk 1]
```diff
@@ -208,6 +208,57 @@ var _ = Describe("The riff function create command", func() {
 			Expect(stdout.String()).To(Equal(fnCreateDryRun))
 		})
 
+		It("should display the nondefault namespace in status hint", func() {
```
Comment (trisberg) [inline cmd/commands/function_test.go]: this description looks identical to the next test although they test different conditions

Comment (fbiville) [reply cmd/commands/function_test.go]: thanks, that's an error, fixing it now

Comment (fbiville) [reply cmd/commands/function_test.go]: Fixed in 3ef092e9a52ba976d30db126cbdd5f8d8497f3e1

**Comment by fbiville** [comment]
Just for information, I added the requested changes via `commit --fixup`, all's needed to do to squash them is `rebase --autosquash [...]`.

**Pull request merged**